app/autotranslate/server/deeplTranslate.js

```diff
@@ -150,7 +150,7 @@ class DeeplAutoTranslate extends AutoTranslate {
 					translations[language] = this.deTokenize(Object.assign({}, message, { msg: translatedText }));
 				}
 			} catch (e) {
-				SystemLogger.error('Error translating message', e);
+				logger.deepl.error('Error translating message', e);
```

Comment:
Interesting! what's the effect of `.deepl.`?

Reply:
using sections will print the logger name on the console. It easy to identify from which provider the exception is raised.

Reply:
see `app/autotranslate/server/logger.js`